include/mapcache.h

```diff
@@ -51,6 +51,10 @@
 #include <apr_time.h>
 
 
+#ifdef USE_REDIS
+#include <hiredis/hiredis.h>
+#endif
+
```

Comment:
For consistency, I would exclude these lines from mapcache.h. On other caches, this optional include is located in the corresponding cache_{}.c file.